### DISCUSSION TITLE
Fallback options?

### DISCUSSION THREAD

**Discussion by bquangDinh** [muxinc/next-video#321] [Q&A]
Hi, I've read the documentation but it seems like that we don't have fallback options in case the browser does not support the format of the video. How can I provide fallback options?

**Comment by luwes** [comment]
which video provider and video formats are you using?

**Comment by bquangDinh** [comment]
I just store it on my hosting server. The format I'm using is AV1.  I just wanted to be sure

**Comment by luwes** [comment]
ah yes, best to use the player only then with `<source>` tags. similar to 
https://next-video-demo.vercel.app/source-tag

you'd probably need to add a fallback to h.264 or so to support all browsers.

source code:
https://github.com/muxinc/next-video/blob/main/examples/default-provider/app/(default)/source-tag/page.tsx

**Comment by bquangDinh** [comment]
Could I use it with BackgroundVideo? The video I used is just a background

**Comment by luwes** [comment]
yes, I think it should work

**Reply by merodiro** [reply]
It didn't work with `BackgroundPlayer`. it added the `source` tag in a `div` outside the `video`
